hook.js

```diff
@@ -80,79 +95,46 @@ function getConfig(repo, branch) {
 
 function getPullRequestCommits(repo, number) {
     var deferred = Q.defer();
+
     var prClient = githubClient.pr(repo, number);
 
     prClient.commits(function(error, commits) {
+        var promises = [];
+
         if (error) {
-            deferred.reject('Fetching commits failed: ' + error);
+            deferred.reject('Error fetching pull request commits:', error);
```

Comment:
Do we want to add more info here, like the project or PR, just for the sake of future debugging?